docs/guide/search/criteria_reference/sibling_criterion.md

```diff
@@ -0,0 +1,15 @@
+# Sibling Criterion
+
+The [`Sibling` Search Criterion](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/API/Repository/Values/Content/Query/Criterion/Sibling.php)
+searches for content under the same parent as the indicated Location.
+
+## Arguments
+
+- `locationId` - int representing the Location ID.
+- `parentLocationId` - int representing the parent Location ID.
+
+## Example
+
+``` php
+$query->query = new Criterion\Sibling(59, 2);
```

Comment:
Instantiation using constructor is correct but in most cases better way is to use named constructor
`\eZ\Publish\API\Repository\Values\Content\Query\Criterion\Sibling::from Location` 

```
$location = $locationService->loadLocation(59);
// ...
$query->query = Criterion\Sibling::fromLocation($location);
```

Reply:
Is it the only Criterion this applies to? The example was written this way to be consistent with other Criteria ref pages.

Reply:
> Is it the only Criterion this applies to?
Currently yes